internal/service/codebuild/webhook.go

```diff
@@ -173,6 +182,7 @@ func resourceWebhookRead(ctx context.Context, d *schema.ResourceData, meta inter
 	}
 
 	d.Set("build_type", webhook.BuildType)
+	d.Set("manual_creation", webhook.ManualCreation)
```

Comment:
We're starting to patch the AWS provider based on this PR and use a local copy to work around this issue. During testing, we found that when this patch is used to update an existing Codebuild project, permadrift is introduced since `ManualCreation` is only valid during webhook creation. The value for `ManualCreation` retrieved by the AWS API is `false` for webhooks existing before the Codebuild API behavior changed.

Handling create-only attributes has examples elsewhere in the AWS provider, like here: https://github.com/hashicorp/terraform-provider-aws/blob/main/internal/service/logs/group.go#L157

My suggested edit is based on that approach, and I've confirmed it resolves the permadrift issue on existing webhooks.

```suggestion
	// Support in-place update of create-only attribute.
	d.Set("manual_creation", d.Get("manual_creation"))
```

There may be a better approach than this.. if there is, hopefully someone chimes in. 😄

Reply:
hey @phuhung273 - do you have time to review and possibly incorporate this change into your PR? we're really hoping your fix can get moved along so we don't have to keep using a patched version of the provider. :) Thanks for your work on this!

Reply:
Thanks @avidspartan1 for supporting, i am honestly ready to do that. But I think we need maintainer help here. The PR has been there for months but no engagement beside from you and me, Im not sure if it will be reviewed or not.